src/ert/run_models/everest_run_model.py

```diff
@@ -437,6 +426,8 @@ def run_experiment(
         self._result = OptimalResult.from_seba_optimal_result(
             seba_storage.get_optimal_result()  # type: ignore
         )
+        optimal_result_from_everstorage = self.ever_storage.get_optimal_result()
+        assert self._result == optimal_result_from_everstorage
```

Comment:
(This is temporary/sanity check to be removed. If all tests pass with this, it means the storage is producing the same optimal results as the old. )